### ISSUE TITLE
Btd6 helper For v34 

### ISSUE THREAD

**Issue by DrollerCandy** [gurrenm3/BTD-Mod-Helper#96]
TL:DR  [10:07:02.040] [ERROR] System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.

//someone resolved a similar issue 

Resolved the error by changing the C++/CLI properties from visual studio project properties , which was referring to .NET framework earlier and i have changed the property as .NET core runtime(clr:netcore) and it is internally handled all GAC changes and worked well.

//Full log file

[10:06:51.460] ------------------------------
[10:06:51.461] MelonLoader v0.5.7 Open-Beta
[10:06:51.462] OS: Windows 10
[10:06:51.462] Hash Code: 49999848531024954985654485098975250521005157544948501005510199495356
[10:06:51.462] ------------------------------
[10:06:51.467] Game Type: Il2Cpp
[10:06:51.468] Game Arch: x64
[10:06:51.468] ------------------------------
[10:06:51.468] Core::BasePath = H:\Epic\BloonsTD6
[10:06:51.469] Game::BasePath = H:\Epic\BloonsTD6
[10:06:51.469] Game::DataPath = H:\Epic\BloonsTD6\BloonsTD6_Data
[10:06:51.470] Game::ApplicationPath = H:\Epic\BloonsTD6\BloonsTD6.exe

[10:06:51.779] ------------------------------
[10:06:51.780] Game Name: BloonsTD6-Epic
[10:06:51.781] Game Developer: Ninja Kiwi
[10:06:51.787] Unity Version: 2021.3.12f1
[10:06:51.787] Game Version: 34.3
[10:06:51.788] ------------------------------

[10:06:52.154] Preferences Loaded!
[10:06:52.160] [Il2CppUnityTls] Patching mono_unity_get_unitytls_interface...
[10:06:52.160] [Il2CppUnityTls] Patching il2cpp_unity_install_unitytls_interface...
[10:06:52.352] [Il2CppUnityTls] Calling Il2CppInstallUnityTlsInterface...

[10:06:52.369] Loading Plugins from 'H:\Epic\BloonsTD6\Plugins'...
[10:06:52.371] 0 Plugins loaded.

[10:06:53.164] [MelonStartScreen] Initializing...
[10:06:53.202] [MelonStartScreen] Using Start Screen Theme: "Default"
[10:06:54.144] Loading Il2CppAssemblyGenerator...
[10:06:54.187] [Il2CppAssemblyGenerator] Contacting RemoteAPI...
[10:06:54.709] [Il2CppAssemblyGenerator] Game Not Found on RemoteAPI Host (https://api.melonloader.com/api/v1/game/bloonstd6-epic)
[10:06:54.712] [Il2CppAssemblyGenerator] RemoteAPI.DumperVersion = null
[10:06:54.713] [Il2CppAssemblyGenerator] RemoteAPI.UnhollowerVersion = null
[10:06:54.714] [Il2CppAssemblyGenerator] RemoteAPI.ObfuscationRegex = null
[10:06:54.715] [Il2CppAssemblyGenerator] RemoteAPI.MappingURL = null
[10:06:54.715] [Il2CppAssemblyGenerator] RemoteAPI.MappingFileSHA512 = null
[10:06:54.717] [Il2CppAssemblyGenerator] Using Dumper Version: 2022.1.0-pre-release.8
[10:06:54.718] [Il2CppAssemblyGenerator] Using Il2CppAssemblyUnhollower Version = 0.4.18.0
[10:06:54.719] [Il2CppAssemblyGenerator] Using Unity Dependencies Version = 2021.3.12
[10:06:54.719] [Il2CppAssemblyGenerator] Using Deobfuscation Regex = null
[10:06:54.720] [Il2CppAssemblyGenerator] Cpp2IL is up to date.
[10:06:54.721] [Il2CppAssemblyGenerator] Il2CppAssemblyUnhollower is up to date.
[10:06:54.722] [Il2CppAssemblyGenerator] UnityDependencies is up to date.
[10:06:54.722] [Il2CppAssemblyGenerator] Checking GameAssembly...
[10:06:55.115] [Il2CppAssemblyGenerator] Assembly Generation Needed!
[10:06:55.121] [Il2CppAssemblyGenerator] Executing Cpp2IL...
[10:06:55.124] [Il2CppAssemblyGenerator] "H:\Epic\BloonsTD6\MelonLoader\Dependencies\Il2CppAssemblyGenerator\Cpp2IL\Cpp2IL.exe" --game-path "H:\Epic\BloonsTD6" --exe-name "BloonsTD6" --use-processor attributeinjector --output-as dummydll
[10:06:55.183] [Il2CppAssemblyGenerator] ===Cpp2IL by Samboy063===
[10:06:55.187] [Il2CppAssemblyGenerator] A Tool to Reverse Unity's "il2cpp" Build Process.
[10:06:55.190] [Il2CppAssemblyGenerator] Version 2022.1.0-pre-release.8
[10:06:55.191] [Il2CppAssemblyGenerator] 
[10:06:55.193] [Il2CppAssemblyGenerator] [Warn] [Program] NO_COLOR set, disabling ANSI color codes as you requested.
[10:06:55.194] [Il2CppAssemblyGenerator] [Info] [Program] Running on Win32NT
[10:06:55.269] [Il2CppAssemblyGenerator] [Info] [Plugins] Loading plugins from H:\Epic\BloonsTD6\MelonLoader\Dependencies\Il2CppAssemblyGenerator\Cpp2IL\Plugins...
[10:06:55.284] [Il2CppAssemblyGenerator] [Info] [Plugins] Using Plugin: Cpp2IL Built-In
[10:06:55.289] [Il2CppAssemblyGenerator] [Info] [Program] Determined game's unity version to be 2021.3.12a0
[10:06:55.329] [Il2CppAssemblyGenerator] [Info] [Library] Initializing Metadata...
[10:06:55.330] [Il2CppAssemblyGenerator] [Info] [Library] 	Using actual IL2CPP Metadata version 29
[10:06:55.500] [Il2CppAssemblyGenerator] [Info] [Library] Initialized Metadata in 172ms
[10:06:55.502] [Il2CppAssemblyGenerator] [Info] [Library] Using binary type Portable Executable (from LibCpp2IL)
[10:06:55.503] [Il2CppAssemblyGenerator] [Info] [Library] Searching Binary for Required Data...
[10:06:55.850] [Il2CppAssemblyGenerator] [Info] [Library] Got Binary codereg: 0x1825F32D0, metareg: 0x182A9A9B0 in 348ms.
[10:06:55.850] [Il2CppAssemblyGenerator] [Info] [Library] Initializing Binary...
[10:06:55.989] [Il2CppAssemblyGenerator] [Info] [Library] Initialized Binary in 139ms
[10:06:56.063] [Il2CppAssemblyGenerator] [Info] [Library] Mapping pointers to Il2CppMethodDefinitions...Processed 114592 OK (74ms)
[10:06:56.071] [Il2CppAssemblyGenerator] [Info] [Program] Creating application model...
[10:06:56.629] [Il2CppAssemblyGenerator] [Warn] [Program] StartOfNextFunc returned va 0xD625A000D6250, raw address 0x0, for raw address 0x24ED720. It should be more than raw address. Falling back to manual, slow, decompiler-based approach.
[10:06:56.688] [Il2CppAssemblyGenerator] [Warn] [Program] StartOfNextFunc returned va 0xD625A000D6250, raw address 0x0, for raw address 0x24ED720. It should be more than raw address. Falling back to manual, slow, decompiler-based approach.
[10:06:56.909] [Il2CppAssemblyGenerator] [Info] [Program] Application model created in 836.7596ms
[10:06:56.910] [Il2CppAssemblyGenerator] [Info] [Program] Pre-processing processing layers...
[10:06:56.911] [Il2CppAssemblyGenerator] [Info] [Program]     Attribute Injector...
[10:06:56.911] [Il2CppAssemblyGenerator] [Info] [Program]     Attribute Injector finished in 0ms
[10:06:56.912] [Il2CppAssemblyGenerator] [Info] [Program] Invoking processing layers...
[10:06:56.913] [Il2CppAssemblyGenerator] [Info] [Program]     Attribute Injector...
[10:06:57.695] [Il2CppAssemblyGenerator] [Info] [Program]     Attribute Injector finished in 784.7128ms
[10:06:57.697] [Il2CppAssemblyGenerator] [Info] [Program] Outputting as Stub ("Dummy") DLL Files to H:\Epic\BloonsTD6\MelonLoader\Dependencies\Il2CppAssemblyGenerator\Cpp2IL\cpp2il_out...
[10:07:00.314] [Il2CppAssemblyGenerator] [ERROR] 
[10:07:00.322] [Il2CppAssemblyGenerator] [ERROR] Unhandled Exception: System.AggregateException: One or more errors occurred. ---> System.NotSupportedException: Object elements in a custom attribute signature should be either 'null' or an instance of BoxedArgument.
[10:07:00.322] [Il2CppAssemblyGenerator] [ERROR]    --- End of inner exception stack trace ---
[10:07:00.322] [Il2CppAssemblyGenerator] [ERROR]    at System.Linq.Parallel.QueryTaskGroupState.QueryEnd(Boolean userInitiatedDispose)
[10:07:00.323] [Il2CppAssemblyGenerator] [ERROR]    at System.Linq.Parallel.DefaultMergeHelper`2.System.Linq.Parallel.IMergeHelper<TInputOutput>.Execute()
[10:07:00.323] [Il2CppAssemblyGenerator] [ERROR]    at System.Linq.Parallel.MergeExecutor`1.Execute[TKey](PartitionedStream`2 partitions, Boolean ignoreOutput, ParallelMergeOptions options, TaskScheduler taskScheduler, Boolean isOrdered, CancellationState cancellationState, Int32 queryId)
[10:07:00.323] [Il2CppAssemblyGenerator] [ERROR]    at System.Linq.Parallel.PartitionedStreamMerger`1.Receive[TKey](PartitionedStream`2 partitionedStream)
[10:07:00.324] [Il2CppAssemblyGenerator] [ERROR]    at System.Linq.Parallel.UnaryQueryOperator`2.UnaryQueryOperatorResults.GivePartitionedStream(IPartitionedStreamRecipient`1 recipient)
[10:07:00.324] [Il2CppAssemblyGenerator] [ERROR]    at System.Linq.Parallel.QueryOperator`1.GetOpenedEnumerator(Nullable`1 mergeOptions, Boolean suppressOrder, Boolean forEffect, QuerySettings querySettings)
[10:07:00.325] [Il2CppAssemblyGenerator] [ERROR]    at System.Linq.Parallel.QueryOpeningEnumerator`1.OpenQuery()
[10:07:00.325] [Il2CppAssemblyGenerator] [ERROR]    at System.Linq.Parallel.QueryOpeningEnumerator`1.MoveNext()
[10:07:00.325] [Il2CppAssemblyGenerator] [ERROR]    at System.Linq.ParallelEnumerable.ToList[TSource](ParallelQuery`1 source)
[10:07:00.326] [Il2CppAssemblyGenerator] [ERROR]    at Cpp2IL.Core.CorePlugin.AsmResolverDummyDllOutputFormat.DoOutput(ApplicationAnalysisContext context, String outputRoot) in C:\Users\Sam\RiderProjects\Cpp2IL\Cpp2IL.Core\CorePlugin\AsmResolverDummyDllOutputFormat.cs:line 88
[10:07:00.329] [Il2CppAssemblyGenerator] [ERROR]    at Cpp2IL.Program.MainWithArgs(Cpp2IlRuntimeArgs runtimeArgs) in C:\Users\Sam\RiderProjects\Cpp2IL\Cpp2IL\Program.cs:line 436
[10:07:00.329] [Il2CppAssemblyGenerator] [ERROR]    at Cpp2IL.Program.Main(String[] args) in C:\Users\Sam\RiderProjects\Cpp2IL\Cpp2IL\Program.cs:line 359

[10:07:01.987] Loading Mods from 'H:\Epic\BloonsTD6\Mods'...
[10:07:01.988] ------------------------------
[10:07:02.033] Melon Assembly loaded: '.\Mods\Btd6ModHelper.dll'
[10:07:02.033] SHA256 Hash: '87d9ea985d5750f663c3a86ad22210e557bdf48e8ab5fe44f3d83236d2346607'
[10:07:02.040] [ERROR] System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.

File name: 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'

  at (wrapper managed-to-native) System.MonoCustomAttrs.GetCustomAttributesInternal(System.Reflection.ICustomAttributeProvider,System.Type,bool)

  at System.MonoCustomAttrs.GetCustomAttributesBase (System.Reflection.ICustomAttributeProvider obj, System.Type attributeType, System.Boolean inheritedOnly) [0x00013] in <e1319b7195c343e79b385cd3aa43f5dc>:0 

  at System.MonoCustomAttrs.GetCustomAttributes (System.Reflection.ICustomAttributeProvider obj, System.Type attributeType, System.Boolean inherit) [0x00037] in <e1319b7195c343e79b385cd3aa43f5dc>:0 

  at System.Reflection.Assembly.GetCustomAttributes (System.Type attributeType, System.Boolean inherit) [0x00000] in <e1319b7195c343e79b385cd3aa43f5dc>:0 

  at System.Attribute.GetCustomAttributes (System.Reflection.Assembly element, System.Boolean inherit) [0x00014] in <e1319b7195c343e79b385cd3aa43f5dc>:0 

  at MelonLoader.MelonUtils.PullAttributesFromAssembly[T] (System.Reflection.Assembly asm, System.Boolean inherit) [0x00000] in <6425afb6ac6a429aaf770255d2f7de57>:0 

  at MelonLoader.MelonUtils.PullAttributeFromAssembly[T] (System.Reflection.Assembly asm, System.Boolean inherit) [0x00000] in <6425afb6ac6a429aaf770255d2f7de57>:0 

  at MelonLoader.MelonAssembly.LoadMelons () [0x0008c] in <6425afb6ac6a429aaf770255d2f7de57>:0 

  at MelonLoader.MelonHandler.LoadMelonsFromDirectory[T] (System.String path) [0x000aa] in <6425afb6ac6a429aaf770255d2f7de57>:0 

  at MelonLoader.Core.Start () [0x0000f] in <6425afb6ac6a429aaf770255d2f7de57>:0
